reciprocalspaceship/dataset.py

```diff
@@ -664,6 +664,25 @@ def label_absences(self, inplace=False):
         self['ABSENT'] = is_absent(self.get_hkls(), self.spacegroup)
         return self
 
+    @inplace
+    def remove_absences(self, inplace=False):
+        """
+        Remove systematically absent reflections in DataSet. 
+
+        Parameters
+        ----------
+        inplace : bool
+            Whether to add the column in place or to return a copy
+
+        Returns
+        -------
+        DataSet
+        """
+        mask = is_absent(self.get_hkls(), self.spacegroup)
+        idx = self.index[mask]
+        self.drop(index=idx, inplace=True)
```

Comment:
Is this intended?  Doesn't this function always operate as if `inplace=True`?

Reply:
i don't know what the antecedent of this is in your second sentence. if you are referring to `pd.DataFrame.drop` the answer is that it does take an `inplace` argument which defaults to `False`. if you are referring to `rs.DataSet.remove_absences`, the answer is that it only operates `inplace` if `inplace=True`. Otherwise the `@inplace` decorator creates a copy before calling the method. 

Does that answer your question?

Reply:
yeah -- i had forgotten whether the `@inplace` decorator had already handled making the copy